### PR TITLE
soundconverter: update to 4.0.3

### PR DESCRIPTION
**Pull request by intrnl**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/soundconverter/template [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-revision=2
+version=4.0.3
+revision=1
 build_style=python3-module
 pycompile_dirs="/usr/lib/${pkgname}/python/${pkgname}"
 hostmakedepends="glib intltool pkg-config python3-gobject python3-distutils-extra"
 makedepends="gtk+3-devel gstreamer1-devel"
 depends="desktop-file-utils gst-plugins-good1 gst-plugins-ugly1
- hicolor-icon-theme python3-gobject"
+ hicolor-icon-theme python3-gobject python3-setuptools"
```
Comment (paper42) [inline srcpkgs/soundconverter/template:10]: gtk+3 seems to be missing

Comment (intrnl) [reply srcpkgs/soundconverter/template:10]: added

**Comment by paper42** [comment]
This PR should contain only one commit, squash them please.

**Comment by intrnl** [comment]
squashed

**Comment by paper42** [comment]
CI is now failing on a conflict, this file shouldn't be in either one of them, I will fix xdaliclock and please `rm` it in `post_install()` like it's done in gnucash.
```
 soundconverter and xdaliclock conflict for
  /usr/share/glib-2.0/schemas/gschemas.compiled
```

**Comment by intrnl** [comment]
added `post_install` hook to remove said file

**Comment by intrnl** [comment]
I'm not sure what's going on, but it seems that if I try to build soundconverter now, on top of latest, I get thrown into having to rebuild practically all other Python packages as well

**Comment by paper42** [comment]
> I'm not sure what's going on, but it seems that if I try to build soundconverter now, on top of latest, I get thrown into having to rebuild practically all other Python packages as well

Either base your branch on the commit before the python 3.11 update, wait until the Python 3.11 build is done or let xbps-src build the required packages for you. Either way, CI will be failing currently because python 3.11's tests are broken and there is a limit on how old the branch can be for CI to run. We won't be merging anything until it's done and when it is, we will trigger CI again.

**Pull request merged**